app/components/Modals/ReceiveModal/ReceiveModal.jsx

```diff
@@ -18,22 +24,33 @@ type Props = {
 }
 
 type State = {
-  imgUri: string,
-  loading: boolean
+  loading: boolean,
+  copied: boolean
 }
 
 export default class ReceiveModal extends React.Component<Props, State> {
   image: ?HTMLImageElement
 
   state = {
-    imgUri: '',
     loading: true
   }
 
+  state = {
+    loading: true,
+    copied: false
+  }
+
+  handleCopyIcon = () => {
+    this.setState({ copied: true })
```

Comment:
this is ok, but I would put it in the callback after copied: true is complete just in case it takes some time to run

Reply:
👍